chaincode-labs/chaincode-podcast/package-relay.md

```diff
@@ -22,652 +22,615 @@ additional_resources:
+So I would like to credit, I think, either BlueMatt, Antoine Riard, or t-bast with coining the term "pinning attack," which is a type of censorship attack on unconfirmed transactions, where you use the fact that there's no propagation guarantees and specific limitations in mempool policy across nodes in the network to prevent a transaction from getting into the mempool or getting mined.
 
 ## Why do you work on package relay?
 
-Speaker 1: 00:06:57
+Adam Jonas: 00:06:57
 
 Cool.
 What attracted you to this project?
 Why this?
 You've made a ton of progress for someone in their first year of Bitcoin Core development.
 
-Speaker 3: 00:07:05
+Gloria Zhao: 00:07:05
 
 Thank you.
 
-Speaker 1: 00:07:06
+Adam Jonas: 00:07:06
 
-And I think there's a lot of lessons to be learned in terms of how you've done that, because a change of this size is not easy to get people excited about or to sort of navigate the social pieces of getting code merged.
+I think there's a lot of lessons to be learned in terms of how you've done that, because a change of this size is not easy to get people excited about, or to navigate the social pieces of getting code merged.
 But yeah, why this?
 
-Speaker 3: 00:07:22
+Gloria Zhao: 00:07:22
 
 Yeah, good question.
-I think it started from me spending a little bit of time in the mempool code and realizing how freaking cool it is because there's so many interesting trade-offs and security concerns you have to be mindful of.
-And then I very quickly figured out that package relay relies primarily getting a safe mempool validation logic in there first.
-And then you can talk about packages on P2P.
-And then I also figured out that package relay is like super non-controversial.
+I think it started from me spending a little bit of time in the mempool code and realizing how freaking cool it is, because there's so many interesting trade-offs and security concerns you have to be mindful of.
+Then I very quickly figured out that package relay relies on primarily getting a safe mempool validation logic in there first, and then you can talk about packages on P2P.
+
+Then I also figured out that package relay is super non-controversial.
 I've never heard anyone say, we don't need package relay.
+
+Adam Jonas: 00:08:00
+
 How'd you figure that out?
-Because I would just talk to people and they would be like, oh, it would be so nice if we had package relay.
-Cause then we could have XYZ and it's like, oh, we can't do this yet because there's no package relay.
-So this is insecure.
 
-Speaker 1: 00:08:13
+Gloria Zhao: 00:08:01
+
+Because I would just talk to people and they would be like, "Oh, it would be so nice if we had package relay, cause then we could have XYZ," and it's like, "Oh, we can't do this yet because there's no package relay, so this is insecure."
```

Comment:
Quick question: What's your thought process on this type of text to decide to change it to:
> [...] they would be like - "Oh, it would be so nice if we had package relay, cause then we could have XYZ" [...] 
or 
> [...] they would be like, "Oh, it would be so nice if we had package relay, cause then we could have XYZ" [...] 
or 
> [...] they would be like - Oh, it would be so nice if we had package relay, cause then we could have XYZ [...] 

I am asking because I've seen examples of all of them in your edits and I was wondering if there is a thought process behind it. That could maybe help us come up with a standardization rule for this kind of scenarios.

Reply:
@kouloumos , you really do have a keen eye, this was an issue that surfaced during this particular transcript.

Here are the nascent thought processes around this issue, still subject to correction and clarification.

The quotative like is used frequently in conversation, and I haven't come to a conclusion as to how to handle these.

The change I made here was to use quotes only if the conversation referred to interactions between one or more parties that happened in the past. If the quotative like is just the descriptive flow, and the speaker is addressing the listeners there and then, I do not think quotes enhance readability in this case anymore. I've done this in the past transcripts, it's just that Gloria tends to use this a lot, so it was never jarring enough.

The punctuation marks ',', '-', and ':'.

The comma is mainly for phrasing, and pauses. The hyphen I find to be mainly useful when a temporary digression is made with a specific example. The colon is for a generalized or complete category that is being explained. (Example - Murch's summary of propagation and confirmation guarantees for unconfirmed transactions)

I'm certain I haven't been consistent with the above explained usage, simply because I'm still trying to figure out what should be done here! At some point, if and when we reach clear guidelines and conclusions about these, I hope to be able to correct these in my past transcripts.